tests/e2e/walletFeatures.spec.ts

```diff
@@ -94,4 +88,31 @@ test.describe("Wallet features", () => {
 
     await browser.close();
   });
+
+  test("auth via LNURL", async () => {
+    const { page, browser, extensionId } = await loadExtension();
+    const $optionsdocument = await unlockExtension({
+      page,
+      extensionId,
+    });
+
+    // go to send
+    await (await findByText($optionsdocument, "Send")).click();
+    const invoiceInput = await findByText(
+      $optionsdocument,
+      "Invoice, Lightning Address or LNURL"
+    );
+    await invoiceInput.type(
+      "lightning:lnurl1dp68gurn8ghj7emfwd6zuemfw3582cn4wdjhycm0de6x2mn59e3k7mf0v4ekxctsv4jxxct59a3rqvnyx4jnvenr8ycxzvfk8qex2vp5xdnrjvty8pjnjvpjv56nvtmjv9mj7v3c8pjxywp3xd3ryvnyxy6rydmyxumxve3exccnxvpcxpnxyvnyx43k2wrpx4skvtmvde6hymrpw46xstt9xfjj6ar9wd6zu6nndahqts4jlx"
+    );
+    await (await getByText($optionsdocument, "Continue")).click();
+
+    await findByText(
+      $optionsdocument,
+      "Do you want to login to gist.githubusercontent.com?"
+    );
+    await findByText($optionsdocument, "Login");
```

Comment:
```suggestion
    await findByText($optionsdocument, "lnurl.getalby.com");
    await findByText($optionsdocument, "Login");
```

I actually think we should also test that the actual login url is shown.

Reply:
Done via `findAllByText` because we have 2 occurrences